user-image/Dockerfile

```diff
@@ -28,9 +28,18 @@ RUN apt-get update && \
 	    # for soc-130ac mapping module
 	    curl \
             # for phys 151
-            gfortran && \
-    apt-get purge && apt-get clean
+            gfortran
+
+# for cogneuro connector, adding NeuroDebian repo
+RUN wget -O- http://neuro.debian.net/lists/zesty.us-ca.libre | tee /etc/apt/sources.list.d/neurodebian.sources.list
+RUN apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80 0xA5D32F012649A5A9
 
+RUN apt-get install --yes
```

Comment:
You need to do an 'apt-get update' before this if you want the packages from the newly added repo to be available to apt.